### DISCUSSION TITLE
can't open prompt by click on dropdown menu or win; shortcut

### DISCUSSION THREAD

**Discussion by codenoobforreal** [johnlindquist/kit#1527] [Q&A]
system: apple m1 pro ventura 13.6.3
download and setup successfully
![image](https://github.com/user-attachments/assets/43371851-d60c-43ee-8962-2883e97bae56)

the open prompt command in the dropdown menu does nothing when I click.

the app shows up and goes away quickly when I use shortcut.If I repeat a couple of times It won't show up anymore.But I still can open dropdown menu.

![image](https://github.com/user-attachments/assets/d92a9490-1959-420e-8ea1-153517bb9629)

any solutions?
![image](https://github.com/user-attachments/assets/f8412281-66ba-4967-9b5b-af77aac615cb)

**Comment by johnlindquist** [comment]
Can you try the v3 beta: https://github.com/script-kit/app/releases/tag/v3.1.12

it looks like you installed from homebrew. I didn't set that up and I'm not sure how it works. If want help debugging 2.3, I'd need the `~/Library/Logs/Kit/main.log`
